snippets/csharp/api/system/span.enumerator/nosync/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading;
+
+class Program
+{
+    private static readonly byte[] _array = new byte[5];
+
+    static void Main()
+    {
+        new Random(42).NextBytes(_array);
+        Span<byte> span = _array;
+
+        Thread thread = new Thread(delegate ()
+        {
+           ClearContents();
+        });
+
+        thread.Start();
```

Comment:
Is using `Thread` API instead of `Task` intentional? Can we replace lines 13...18 with:
```csharp
Task.Run(ClearContents);
```

I understand we cannot make `async Main` and we cannot launch the span enumeration task. Also we might not want to use `Task.Delay(20).Wait()` instead of `Thread.Sleep(20)`. Thus, using `Thread` API for consistency everywhere.

Reply:
I think we could use `async Main`, that's a released feature.

Reply:
Good point. That's what I get for reviewing before :coffee: